Pipfile

```diff
@@ -6,8 +6,9 @@ name = "pypi"
 [packages]
 "72eb2aa" = {file = "https://github.com/Rapptz/discord.py/archive/rewrite.zip"}
 aiodns = "*"
-aiohttp = "<2.3.0,>=2.0.0"
+aiohttp = "*"
```

Comment:
any particular reason this was done? discord.py doesn't like the latest aiohttp, we locked the version for a reason.